### PR TITLE
[aon_timer,dv] DV Document Updates

### PR DESCRIPTION
**Pull request by ctopal**
Also includes a minor naming fix in a coverage point

### CODE REVIEW

**Review by cindychip — APPROVED** [review]

File: hw/ip/aon_timer/doc/dv/index.md [hunk 1]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
 #### Test sequences
 
 The test sequences can be found in `hw/ip/aon_timer/dv/env/seq_lib`.
-The basic test virtual sequence configures the block in a valid initial state, but doesn't enable either timer.
+The basic test virtual sequence (aon_timer_base_vseq) configures the block in a valid initial state, but doesn't enable either timer.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:88]: Nit: I'd probably use back-ticks to put the code name in a monospace font.

File: hw/ip/aon_timer/doc/dv/index.md [hunk 2]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
 This is used as a base class for automated tests like the CSR tests.
 
-<div class="bd-callout bd-callout-warning">
+Smoke sequence initializes both timers, starts them with random threshold values configured.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:91]: Grammar nit: This needs an article, so "The smoke sequence..." and you've got a two element list of things, so it needs to be "... both timers and starts them with ...".

File: hw/ip/aon_timer/doc/dv/index.md [hunk 3]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
+After that it waits until an interrupt is catched and clears everything before shutting down.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:92]: Nit: Should be "interrupt is caught" (or maybe "seen"?)

File: hw/ip/aon_timer/doc/dv/index.md [hunk 4]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
 
-**TODO**: Describe other test sequences here.
+Lock sequence tries to change the configuartion of the watchdog time while it is locked by software.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:94]: Again, this should probably be "The lock sequence". Also, typo in "configuration". And do you mean "watchdog time**r**"?

File: hw/ip/aon_timer/doc/dv/index.md [hunk 5]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
 
-<div class="bd-callout bd-callout-warning">
+To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
+The following cover points have been developed to prove that the test intent has been adequately met.
 
-**TODO**: Functional coverage points are not yet defined.
+`prescale_cp`: This cover point includes possible values for the prescale register of wakeup timer.
 
-</div>
+`bark_thold_cp`: This cover point includes bark thershold configurations of watchdog timer.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:105]: Typo in "threshold"

File: hw/ip/aon_timer/doc/dv/index.md [hunk 6]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
+Stress sequence tries to change all configurations while running.
 
 #### Functional coverage
 
-<div class="bd-callout bd-callout-warning">
+To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
+The following cover points have been developed to prove that the test intent has been adequately met.
 
-**TODO**: Functional coverage points are not yet defined.
+`prescale_cp`: This cover point includes possible values for the prescale register of wakeup timer.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:103]: Nit: I'd consider removing the "This cover point" prefix from each of these items.

File: hw/ip/aon_timer/doc/dv/index.md [hunk 7]
```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
...
 
-</div>
+`bark_thold_cp`: This cover point includes bark thershold configurations of watchdog timer.
+
+`bite_thold_cp`: This cover point includes bite threshold configurations of watchdog timer.
+
+`wkup_thold_cp`: This cover point includes threshold configurations of wakeup timer.
+
+`wkup_cause_cp`: This cover point is for checking if we ever clear wakeup interrupt by writing to `WKUP_CAUSE` register.
```
Comment (rswarbrick) [inline hw/ip/aon_timer/doc/dv/index.md:111]: Nit: Should be "clear **the** wakeup interrupt"

**Review by rswarbrick — APPROVED** [review]

**Pull request merged**